ONSdigital/sdc-int-rh-ui
# 681 add in welsh content to respondent home

# Motivation and Context


Added translated Welsh content

# What has changed




Added a make command for quick compiling translation files. 
Fixed the back button at the top of the page, which was always redirecting to the en page.
Added translated text to the .po files, config, and tests.
Updated the tests.
Removed some unused variables and left some comments on unused code. 

Note: 
There are still some content that needs to be translated, I'll add a card for that.

# How to test?



Run tests, open the welsh version of the page check that there's no placeholders.
(Apart from "The following links open in a new tab" and the error message when JS in the browser is disabled as the translation for that still needs to be done)

# Links



https://trello.com/c/KmJJQQz8/681-add-in-welsh-content-to-respondent-home-5-kp

# Screenshots (if appropriate):
